core/mobile/src/backend/MobileRpcServer.ts

```diff
@@ -143,6 +143,7 @@ export function setupMobileRpc() {
   }
 
   let server: MobileRpcServer | null = new MobileRpcServer();
+  let retainUvLoop: NodeJS.Timer | undefined;
```

Comment:
I think there should be a comment explaining why this exists.

Reply:
Agreed. I'll add one.